code/src/java/pcgen/output/model/InfoModel.java

```diff
@@ -78,21 +78,22 @@ public TemplateModel get(String key) throws TemplateModelException
 	{
 		CaseInsensitiveString cis = new CaseInsensitiveString(key);
 		MessageFormat info = cdo.get(MapKey.INFO, cis);
-		if (info == null)
+		
+		StringBuffer sb = new StringBuffer(100);
+		if (info != null)
```

Comment:
I'm not a fan here.  If info is null, we need to produce some form of error... not suppress it and leave it in the output sheet.